packages/react/src/components/Storage/FileUploader/UploadDropZone/index.tsx

```diff
@@ -13,5 +13,19 @@ export function UploadDropZone({
   onDrop,
   onDragOver,
 }: UploadDropZoneProps): JSX.Element {
-  return <Card>{children}</Card>;
+  return (
+    <Card
+      className={classNames(inDropZone && 'active', 'amplify-fileuploader')}
```

Comment:
`inDropZone && 'active'` will resolve to `false` right? How does that work with  `classNames`?

Reply:
When the `inDropZone` variable is true, it adds the `active` class. It seems to work.